intelmq/bots/collectors/bitsight/collector.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+import re
+import sys
+import pycurl
+from intelmq.lib.bot import Bot
+from intelmq.lib.message import Report
+
+
+class BitsightCollectorBot(Bot):
+
+      def init(self):
+          self.logger.info("Connecting to BitSightTech stream server") 
+          http_proxy = self.parameters.http_proxy 
+          https_proxy = self.parameters.http_ssl_proxy 
+          self.conn  = pycurl.Curl()
+          if http_proxy:
+             self.conn.setopt(pycurl.PROXY, str(http_proxy))
+          if https_proxy:
+             self.conn.setopt(pycurl.PROXY, str(https_proxy))
+          self.conn.setopt(pycurl.URL, str(self.parameters.http_url))
+          self.conn.setopt(pycurl.WRITEFUNCTION, self.on_receive)
+
+      def process(self):
+          self.conn.perform()
+
+      def on_receive(self, data):
+          for line in data.decode().splitlines():
+              line = line.strip()
+              if line == "":
+                  continue
+
+              report = Report()
+              report.add("raw", str(line))
+              report.add("feed.name", self.parameters.feed)
+              report.add("feed.accuracy", self.parameters.accuracy)
+              url = re.sub(r'[kK][eE][yY]=.*', '', str(self.parameters.http_url)) # removing api_key
```

Comment:
The API key is not mine to share :( I don't have permission to do so. Also I think it would be against the contract we have with BitSight.
I just added this to build the test, as it would show on the code.